dwave/system/composites/cutoffcomposite.py

```diff
@@ -27,33 +27,50 @@
 
 
 class CutOffComposite(dimod.ComposedSampler):
-    """Composite to cut off small interactions.
+    """Composite to remove interactions below a specified cutoff value.
 
-    Removes interactions smaller than a given cutoff. Isolated
-    variables (after the cutoff) are also removed.
-
-    Note that if the problem had isolated variables before the cutoff, they
-    will also be affected.
+    Downsizes the binary quadratic model (BQM) submitted to the child sampler by retaining
```

Comment:
To me at least downsizes == scales which I find confusing.

Reply:
I originally had "Reduces" but thought "Downsizes" more accurate and don't see it as synonymous with scaling, where proportionality is maintained, but can happily replace with "Prunes". I actually now prefer prunes.

Reply:
Yeah, I like prunes much more

Reply:
Done